pyproject.toml

```diff
@@ -58,3 +58,6 @@ exclude = "examples"
 
 [tool.ruff]
 line-length = 119
+
```

Comment:
Please remove this config from the toml. This package uses PDM for package distribution. Might have accidentally been placed here?

Reply:
Yes this is because I use flint (not PDM) to share personal package (using fury) so I though it was not annoying.. But if you force me to use PDM, no problem

Reply:
Thanks